microservices-dashboard-server/src/main/java/be/ordina/msdashboard/events/HealthInfoFailed.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2015-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package be.ordina.msdashboard.events;
+
+import org.springframework.cloud.client.ServiceInstance;
+import org.springframework.context.ApplicationEvent;
+
+/**
+ * An applicationEvent which signals a failure while retrieving HealthInfo.
+ *
+ * @author Dieter Hubau
+ */
+public class HealthInfoFailed extends ApplicationEvent {
```

Comment:
Can you rename this to `HealthInfoRetrievalFailed`?
This will make it more clear in the codebase what exactly failed.